spec/lib/krikri/enrichments/split_coordinates_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'spec_helper'
+
+describe Krikri::Enrichments::SplitCoordinates do
+  it_behaves_like 'a field enrichment'
+
+  context 'with place object' do
+    let(:lat) { nil }
+    let(:long) { nil }
+    let(:place) do
```

Comment:
Minor style... maybe collapse this onto one line?

`let(:place) { build(:place, lat: lat, long: long }`